__tests__/app.test.js

```diff
@@ -35,3 +35,39 @@ describe("/topic", () => {
       });
   });
 });
+
+describe("/articles/:article_id", () => {
+  test("GET:200 sends an object containing a single article to the end point ", () => {
+    return req(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.article).toMatchObject({
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 100,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
```

Comment:
I would add the article id here and check that it is 1, that will definitely prove you have the correct article coming back.